### PR TITLE
Add metrics page to supplier frontend and test

### PR DESCRIPTION
**Pull request by benvand**
https://trello.com/c/IOBExody/295-add-custom-python-metrics-to-supplier-frontend

### CODE REVIEW

File: tests/app/main/test_metrics.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# coding=utf-8
+from tests.app.helpers import BaseApplicationTest
+import re
+
+
+def load_prometheus_metrics(response_bytes):
+    return dict(re.findall(b"(\w+{.+?}) (\d+)", response_bytes))
```
Comment (risicle) [inline tests/app/main/test_metrics.py]: :rotating_light: should be `br"...` here so the pattern's interpreted raw :rotating_light:

(or `rb"...` if you want to confuse everyone :gem: )

Comment (risicle) [reply tests/app/main/test_metrics.py]: (wonder if there's a flake8 check for this)

Comment (benvand) [reply tests/app/main/test_metrics.py]: What does the rawness do? Better for regexes?

Comment (benvand) [reply tests/app/main/test_metrics.py]: Happy to do by the wat 😂

Comment (benvand) [reply tests/app/main/test_metrics.py]: *way

Comment (benvand) [reply tests/app/main/test_metrics.py]: Although saying that I'm closing off this PR because I've done it fully in brief responses

**Pull request closed without merging**